hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -594,6 +598,11 @@ public TableDescriptorBuilder setReplicationScope(int scope) {
     return this;
   }
 
+  public TableDescriptorBuilder setRegionServerGroup(String group) {
```

Comment:
Add java doc for the public APIs.

Sorry for late comment.

Reply:
Added